### ISSUE TITLE
Very low FPS and inconsistent frametime when running 3DMigoto

### ISSUE THREAD

**Issue by Slangtooth** [SilentNightSound/GI-Model-Importer#162]
Any time I try to run any mod, regardless if it's something as seemingly simply and light as a character portrait mod, or several full character skin mods, the result is the same. I've tried a wide range of different things in search of a solution. From checking individual mods one by one, to performing a full driver uninstall and reinstall, along with plenty of other things I can go in detail about.

Been troubleshooting with both the Dev and "for playing" versions. This is on a setup that runs the game with zero issue on the highest graphic preset, at a solid 60Frames when not using 3DMigoto, which is more confusing.

**Comment by Doktorishka2** [comment]
[Try this](https://github.com/SilentNightSound/GI-Model-Importer/issues/159)

**Comment by Slangtooth** [comment]
Unfortunately have tried both off and on for Vysnc already. Even tried the opposite route with FPS unlocker just in case.

**Comment by leotorrez** [comment]
is normal to have a slight frame drop when you first load a mod in a play session for its files need to be pushed to your RAM. However if you are losing frames by standing still the issue might be elsewhere

**Comment by Slangtooth** [comment]
Yeah I've tried giving it some time to load before doing anything and will still find it losing frames. I'm still at a bit of a loss as to what could be the cause.

**Comment by leotorrez** [comment]
3dm does consume some resources- you can use ctrl + f9 to check exactly what.
